packages/sdk-codegen/src/go.gen.ts

```diff
@@ -24,276 +24,332 @@
 
  */
 
-import {CodeGen, IVersionInfo} from './codeGen'
+import { commentBlock } from '@looker/sdk-codegen-utils'
+import { CodeGen, IVersionInfo } from './codeGen'
 import {
-    ApiModel, Arg,
-    EnumType,
-    EnumValueType,
-    IMappedType,
-    IMethod,
-    IParameter,
-    IProperty,
-    IType, strBody,
+  ApiModel,
+  Arg,
+  EnumType,
+  EnumValueType,
+  IMappedType,
+  IMethod,
+  IParameter,
+  IProperty,
+  IType,
+  strBody,
```

Comment:
You should be able to set up prettier for your code editor so it will take care of these mechanical reformatting things for you.